cookbooks/ros2_windows/recipes/visual_studio.rb

```diff
@@ -30,7 +30,14 @@
 
 installer_options = base_options + package_arguments
 
-visual_studio_source = 'https://aka.ms/vs/16/release/vs_%s.exe' % node['ros2_windows']['vs_version']
+case node['ros2_windows']['vs_release']
+when '2022'
+  visual_studio_source = 'https://aka.ms/vs/17/release/vs_%s.exe' % node['ros2_windows']['vs_version']
+  visual_studio_path = 'c:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\%s' % vs_version_camel_case
```

Comment:
```suggestion
  visual_studio_source = "https://aka.ms/vs/17/release/vs_#{node['ros2_windows']['vs_version']}.exe"
  visual_studio_path = "c:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\#{vs_version_camel_case}"
```

Interpolated strings are more common than formatted ones.